webpack.config.js

```diff
@@ -0,0 +1,174 @@
+import { CleanWebpackPlugin } from 'clean-webpack-plugin';
+import webpack from 'webpack';
+import VirtualModulesPlugin from 'webpack-virtual-modules';
+import CopyPlugin from 'copy-webpack-plugin';
+import MiniCssExtractPlugin from 'mini-css-extract-plugin';
+import RemovePlugin from 'remove-files-webpack-plugin';
+import fs from 'fs/promises';
+import path from 'path';
+import { createRequire } from 'module';
+import globPkg from 'glob';
+import TerserPlugin from 'terser-webpack-plugin';
+import postcssUrl from 'postcss-url';
+import got from 'got';
+import CssMinimizerPlugin from 'css-minimizer-webpack-plugin';
+
+const git = createRequire(import.meta.url)('git-rev-sync');
+const { EnvironmentPlugin, optimize } = webpack;
+const { glob } = globPkg;
+
+function convertEmojiToolkitCodePointToChar(codePoint) {
+    if (codePoint.includes('-')) {
+        return codePoint.split('-')
+            .map(subCodePoint => convertEmojiToolkitCodePointToChar(subCodePoint))
+            .join('');
+    }
+
+    const charCode = parseInt(codePoint, 16);
+    if (charCode >= 0x10000 && charCode <= 0x10FFFF) {
+        const high = Math.floor((charCode - 0x10000) / 0x400) + 0xD800;
+        const low = ((charCode - 0x10000) % 0x400) + 0xDC00;
+        return (String.fromCharCode(high) + String.fromCharCode(low));
+    }
+
+    return String.fromCharCode(charCode);
+}
+
+function jsonTransform(emojis) {
+    const result = {};
+    for (const emojiData of Object.values(emojis)) {
+        const char = convertEmojiToolkitCodePointToChar(emojiData.code_points.fully_qualified);
+        const data = {
+            char,
+            slug: emojiData.shortname.replace(/:/g, ''),
+        };
+        result[data.slug] = data;
+        for (const alternativeShortName of emojiData.shortname_alternates) {
+            result[alternativeShortName.replace(/:/g, '')] = data;

```

Comment:
you will need to merge in master or rebase your changes off master to pull in recent changes